sipuadart/test/sipuadart_test.dart

```diff
@@ -1,13 +1,137 @@
 import 'package:flutter_test/flutter_test.dart';
-
-import 'package:sipuadart/sipuadart.dart';
+import 'package:sipuadart/sdp/attributes/Category.dart' as sdpCategory;
```

Comment:
I think we can remove the aliasing `as sdpCategory` here and use `Category` in the tests the same way we are doing for the other attributes e.g. `Keywords`. What do you think?

Reply:
There was a conflict with foundation package. But I think I not using this package anymore. So I''ll check it again. Tks